Automattic/woocommerce-payments
# Add welcome tour to fraud protection settings

Fixes Automattic/woocommerce-payments-server#3083

#### Changes proposed in this Pull Request



Add the welcome tour component to the WooCommerce payments settings.







#### Testing instructions





* Checkout to this branch `add/3083-welcome-tour`
* Run `npm start`
* Enable the `wcpay_fraud_protection_settings_active` flag on your client
  * If you don't have it set yet, run: `npm run wp option add wcpay_fraud_protection_settings_active 1` 
  * Otherwise, update it by running: `npm run wp option update wcpay_fraud_protection_settings_active 1`
* Go to **Payments > Settings**
* The welcome tour should pop up
* Go through the tour steps and finish it – or click on the close button
* Refresh the page – it shouldn't pop up again
* Check the welcome tour flag by running `npm run wp option get wcpay_fraud_protection_welcome_tour_dismissed` – it should return `1`
* If you want to test it again, run the following command:  `npm run wp option update wcpay_fraud_protection_welcome_tour_dismissed 0`
* Refresh the page – the tour should pop up again

-------------------


**Post merge**
